### PR TITLE
fix find duplicates in qconfig

### PR DESCRIPTION
**Pull request by eladc-git**
## Pull Request Description:
Use names in is_match method only in tensorflow operations.

## Checklist before requesting a review:
- [x] I set the appropriate labels on the pull request.
- [x] I have added/updated the release note draft (if necessary).
- [x] I have updated the documentation to reflect my changes (if necessary).
- [x] All function and files are well documented. 
- [x] All function and classes have type hints.
- [x] There is a licenses in all file.
- [x] The function and variable names are informative. 
- [x] I have checked for code duplications.
- [x] I have added new unittest (if necessary).

### CODE REVIEW

File: model_compression_toolkit/core/common/graph/base_node.py [hunk 1]
```diff
@@ -553,7 +553,7 @@ def get_qco(self, tpc: TargetPlatformCapabilities) -> QuantizationConfigOptions:
         # Extract qco with is_match_type to overcome mismatch of function types in TF 2.15
         matching_qcos = [_qco for _type, _qco in tpc.layer2qco.items() if self.is_match_type(_type)]
         if matching_qcos:
-            if len(matching_qcos) > 1:
+            if len(matching_qcos) > 1 and matching_qcos[0] != tpc.tp_model.default_qco:
```
Comment (elad-c) [inline model_compression_toolkit/core/common/graph/base_node.py:556]: 1. Must the deafult be at index 0?
2. Add a test to simulate it.

**Review by elad-c — CHANGES_REQUESTED** [review]

File: model_compression_toolkit/core/common/graph/base_node.py (hunk 1 above)
Comment (ofirgo) [inline model_compression_toolkit/core/common/graph/base_node.py:556]: I don't think that this solves the issue correctly.
I suspect that the case generating this issue is when there is a pytorch function that has two aliases, and that one of them appears in a TPC operation set and the other is not, thus returning the default qco.

Please try to reproduce the issue, identify the pytorch layer that has duplicate configs and let's think of a more robust solution.

**Comment by elad-c** [comment]
Issue will be fixed in another PR.
